resources/home/dnanexus/utils/utils.py

```
@@ -41,181 +47,21 @@ def read_dxfile(
-    df.to_excel(writer, sheet_name=sheet_name, index=include_index)
-    worksheet = writer.sheets[sheet_name]
-
-    _set_tab_color(worksheet, tab_color)
-
-    # Add extra columns if provided
-    if extra_cols:
-        _add_extra_columns(worksheet, extra_cols)
-
-    _apply_header_format(worksheet)
-    _adjust_column_widths(worksheet)
+    return {column: DATAFRAME_TYPES[column] for column in columns}
```

Ruff (0.8.2)

Undefined name `DATAFRAME_TYPES`

(F821)